### PR TITLE
162109840 user signup update

### PR DESCRIPTION
**Pull request by cob04**
#### What does this PR do?
* creates a signup endpoint  `POST` for creating a user account
* creates a login endpoint  `POST` for getting an jwt access token

### How should this be manually tested?
* Clone this repo `git clone -b ft-user-signup-update-162109840 https://github.com/cob04/Send-It.git`

#### What are the relevant pivotal tracker stories?
[https://www.pivotaltracker.com/story/show/162109840](https://www.pivotaltracker.com/story/show/162109840)

### CODE REVIEW

File: app/api/version3/views/user_views.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            payload = {
+                "message": "Success",
+                "user": user.to_dict()
+            }
+            return payload, 200
+        else:
+            payload = {
+                "message": "Sorry, we cannot find such a user",
+                "error": "Not found"
+            }
```
Comment (jomomg) [inline app/api/version3/views/user_views.py:50]: Include a 400 status code here.

**Review by jomomg — CHANGES_REQUESTED** [review]

File: app/db_config.py [hunk 2]
```diff
@@ -65,8 +65,9 @@ def create_table_queries():
     users = """CREATE TABLE IF NOT EXISTS users (
         user_id serial PRIMARY KEY NOT NULL,
         name character varying(50) NOT NULL,
-        email character varying(50) NOT NULL,
-        password character varying(500) NOT NULL)"""
+        email character varying(50) UNIQUE NOT NULL,
+        password character varying(500) NOT NULL,
```
Comment (jomomg) [inline app/db_config.py:69]: Why does the password have a 500 character limit?

File: app/api/version3/views/user_views.py (hunk 1 above)
Comment (jomomg) [reply app/api/version3/views/user_views.py:50]: and a return statement

File: app/db_config.py (hunk 2 above)
Comment (cob04) [reply app/db_config.py:69]: I'm going to hash the password and store the hash instead

File: app/api/version3/views/user_views.py (hunk 1 above)
Comment (cob04) [reply app/api/version3/views/user_views.py:50]: will do.

**Pull request merged**